SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/EditorWorkerController.h

```diff
@@ -45,20 +45,30 @@ struct EditorWorkerController
 		FSpatialGDKEditorToolbarModule& Toolbar = FModuleManager::GetModuleChecked<FSpatialGDKEditorToolbarModule>("SpatialGDKEditorToolbar");
 		SpatialShutdownHandle = Toolbar.OnSpatialShutdown.AddRaw(this, &EditorWorkerController::OnSpatialShutdown);
 
-		int32 PlayNumberOfServers;
-		GetDefault<ULevelEditorPlaySettings>()->GetPlayNumberOfServers(PlayNumberOfServers);
-
-		WorkerIds.SetNum(PlayNumberOfServers);
-		for (int i = 0; i < PlayNumberOfServers; ++i)
+		const TMap<FString, int32>& WorkerTypesToLaunch = GetDefault<ULevelEditorPlaySettings>()->WorkerTypesToLaunch;
```

Comment:
Is it possible to get the version of this from launch config that uses `FWorkerTypeLaunchSection` which is a nicer type?

Reply:
Yes I think we will be able to, will update